File: python/demo/karyawan/2-detail-karyawan.py
"""
DEMO: Detail 1 karyawan (data pribadi, karir, payroll)

Jalankan lewat terminal:  python 2-detail-karyawan.py
"""

import sys
from pathlib import Path

sys.path.insert(0, str(Path(__file__).resolve().parents[2]))  # agar gajihub.py bisa di-import

from gajihub import api_get, mulai_demo, tampilkan

mulai_demo('Detail Karyawan')

id_karyawan = 1  # ganti dengan ID karyawan (lihat demo 1-daftar-karyawan.py)

hasil = api_get(f'/hr/employees/{id_karyawan}')

tampilkan(hasil)

File: nodejs/demo/karyawan/2-detail-karyawan.mjs
/**
 * DEMO: Detail 1 karyawan (data pribadi, karir, payroll)
 *
 * Jalankan lewat terminal:  node 2-detail-karyawan.mjs
 */

import { apiGet, mulaiDemo, tampilkan } from '../../gajihub.mjs';

mulaiDemo('Detail Karyawan');

const idKaryawan = 1; // ganti dengan ID karyawan (lihat demo 1-daftar-karyawan.mjs)

const hasil = await apiGet('/hr/employees/' + idKaryawan);

tampilkan(hasil);
